### PR TITLE
Sending direct deposit transactions to the pool

### PR DESCRIPTION
**Pull request by EvgenKor**
- Interact with DD contract to send transactions directly
- Added ability to set gas price coefficient

### CODE REVIEW

File: lib/networks/client.d.ts [hunk 1]
```diff
@@ -24,6 +24,8 @@ export declare abstract class Client {
     approve(tokenAddress: string, spender: string, amount: string): Promise<string>;
     increaseAllowance(tokenAddress: string, spender: string, additionalAmount: string): Promise<string>;
     allowance(tokenAddress: string, spender: string): Promise<bigint>;
+    getDirectDepositContract(poolAddress: string): Promise<string>;
+    directDeposit(poolAddress: string, amount: string, zkAddress: string): Promise<string>;
```
Comment (AllFi) [inline lib/networks/client.d.ts:28]: I guess it should be ddAddress or something like that.

**Review by AllFi — APPROVED** [review]
Looks good to me.

File: lib/networks/client.d.ts (hunk 1 above)
Comment (EvgenKor) [reply lib/networks/client.d.ts:28]: No, here is a Pool contract address passed as a first parameter. DD contract address is fetched under-the-hood

Comment (EvgenKor) [reply lib/networks/client.d.ts:28]: `getDirectDepositContract ` method returns ddAddress to approve tokens to the right address in the console

**Review by k1rill-fedoseev — APPROVED** [review]

File: lib/networks/client.d.ts (hunk 1 above)
Comment (AllFi) [reply lib/networks/client.d.ts:28]: Got it, thanks

**Pull request merged**